ballista/executor/src/circuit_breaker/client.rs

```diff
@@ -315,28 +317,41 @@ impl CircuitBreakerClient {
                         .remove(scheduler_id)
                         .unwrap_or_default();
 
-                    per_scheduler_state
-                        .updates
-                        .entry(update.key)
-                        .and_modify(|percent| {
-                            *percent = update.percent.max(*percent);
-                        })
-                        .or_insert(update.percent);
-
-                    if per_scheduler_state.updates.len() >= config.max_batch_size
-                        || per_scheduler_state.last_sent.add(config.send_interval) < now
+                    if let Some(entry) = per_scheduler_state.updates.get_mut(&update.key)
                     {
-                        Self::process_batch(
+                        *entry = update.percent.max(*entry);
+                    } else if per_scheduler_state.updates.len() >= config.max_batch_size {
+                        // No logging because it would be very spammy, should be easy to detect from the metrics.
```

Comment:
So in this branch the update just gets dropped?

Reply:
Might be good to add a prometheus metric here which we can inc when this happens

Reply:
good idea

Reply:
done.